### PR TITLE
Sysbuil mcuboot conf untangle

### PR DESCRIPTION
**Pull request by nvlsianpu**
This is try to migrate with configuration automatization to Kconfig for MCUBoot configured to use external-crypto (by CONFIG_BOOT_NRF_EXTERNAL_CRYPTO=y) provided by NSIB.

### CODE REVIEW

File: subsys/bootloader/bl_crypto/Kconfig [hunk 1]
```diff
@@ -34,6 +34,7 @@ config SB_HASH_LEN
 
 choice SB_CRYPTO_SIG
 	prompt "Firmware verification Algorithm"
+	default SB_CRYPTO_CLIENT_ECDSA_SECP256R1 if BOOT_USE_NRF_EXTERNAL_CRYPTO
```
Comment (nvlsianpu) [inline subsys/bootloader/bl_crypto/Kconfig:37]: .. & MCUBOOT

**Comment by nvlsianpu** [comment]
Take this into account: https://github.com/nrfconnect/sdk-nrf/pull/15748

**Review by sigvartmh — APPROVED** [review]

File: sysbuild/mcuboot_external_crypto.conf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#
+# CONFIG_BOOT_NRF_EXTERNAL_CRYPTO=y (Use crypto backend from B0)
+#
+# CONFIG_SB_CRYPTO_CLIENT_ECDSA_SECP256R1=y
+# CONFIG_SB_CRYPTO_CLIENT_SHA256=y
+# CONFIG_BL_SHA256_EXT_API_REQUIRED=y
+# CONFIG_BL_SECP256R1_EXT_API_REQUIRED=y
+#
+# What's more, ECDSA is default signing mechanism 
+# CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256=y
```
Comment (tejlmand) [inline sysbuild/mcuboot_external_crypto.conf:24]: newline missing

File: sysbuild/enable_secure_bootloader.conf [hunk 3]
```diff
@@ -2,4 +2,5 @@
 # Bootloader Configuration
 
 CONFIG_SECURE_BOOT=y
-CONFIG_FW_INFO=y
+# Implies by abowe
```
Comment (tejlmand) [inline sysbuild/enable_secure_bootloader.conf:5]:
```suggestion
# Implies by above
```

**Pull request closed without merging**